_events/sr2021/kickstart.md

```diff
@@ -1,14 +1,28 @@
 ---
 title: Kickstart
-date: 2020-11-21
+date: 2020-11-21 13:00:00
```

Comment:
Maybe add a timezone here? Not sure if Jekyll supports that though.

Reply:
[Looks like](https://mehmandarov.com/jekyll-content-on-time/#setting-the-time-zone-in-the-jekyll-configuration-file) Jekyll will assume this is a UTC DateTime and render it using the time zone in our `_config.yml`. Currently our time zone is set to `Etc/GMT` maybe it should be `Europe/London`? This particular datetime is fine as UK time currently matches `Etc/GMT` but I could add an offset of `+00:00` to make it obvious if that would be useful?

Reply:
Yeah, I think a habit of these always having a timezone is a good idea.

Reply:
Maybe leave as-is, and we can address timezone-y bits in a separate PR? Given we do cater to teams and volunteers in multiple timezones, we should probably address it.